VAMobile/src/screens/HomeScreen/ProfileScreen/SettingsScreen/InAppFeedbackScreen/InAppFeedbackScreen.tsx

```diff
@@ -28,8 +29,28 @@ function InAppFeedbackScreen({ navigation }: InAppFeedbackScreenProps) {
   // })
 
   const onSubmit = (): void => {
+    const { found, newText } = checkStringForPII(task)
     // logAnalyticsEvent(Events.vama_feedback_submitted(taskCompleted, satisfaction))
-    navigation.goBack()
+    if (found) {
+      console.log('Before task: ', task)
```

Comment:
probably should clean up these console logs

Reply:
fixed as suggested.